src/components/Progress/Progress.scss

```diff
@@ -62,26 +64,69 @@ $progress-thinnest-height: 4px;
 
         &_theme_default {
             background-color: var(--yc-color-infographics-neutral-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-neutral-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-neutral-medium);
+            }
         }
 
         &_theme_success {
             background-color: var(--yc-color-infographics-positive-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-positive-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-positive-medium);
+            }
         }
 
         &_theme_warning {
             background-color: var(--yc-color-infographics-warning-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-warning-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-warning-medium);
+            }
         }
 
         &_theme_danger {
             background-color: var(--yc-color-infographics-danger-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-danger-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-danger-medium);
+            }
         }
 
         &_theme_info {
             background-color: var(--yc-color-infographics-info-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-info-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-info-medium);
+            }
         }
 
         &_theme_misc {
             background-color: var(--yc-color-infographics-misc-medium);
+
+            &#{$item}_loading {
+                --yc-progress-loading-color-1: var(--yc-color-infographics-misc-light);
+                --yc-progress-loading-color-2: var(--yc-color-infographics-misc-medium);
```

Comment:
Custom colors are not supported. We should you semi-transparent neutral colors for stripes for custom backgrounds